core/rawdb/eradb/eradb.go

```diff
@@ -0,0 +1,345 @@
+// Copyright 2025 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+// Package eradb implements a history backend using era1 files.
```

Comment:
Just wondering how this will be used. Does it show improvements in terms of efficiency in accessing db files?

Reply:
If you prune your node from historical data but then you want to access this historical data for some reason, you can download it as era1 files https://geth.ethereum.org/docs/fundamentals/downloadera 

I think we wont be using this feature atleast for now @manav2401 @pratikspatil024 ?